### ISSUE TITLE
"Found Typo - Create pull request on Github"-link?

### ISSUE THREAD

**Issue by rnarian** [frdmn/blog.frd.mn#19]

**Comment by rnarian** [comment]
There are posts where you suggest this within the post itself:

> ![bildschirmfoto 2016-04-05 um 09 38 47](https://cloud.githubusercontent.com/assets/1174951/14274540/41dc2bda-fb12-11e5-8998-5159b292caac.png)
> (https://github.com/frdmn/blog.frd.mn/blob/master/src/posts/2014-09-15_how-to-set-up-proper-startstop-services-ubuntu-debian-mac-windows.md#windows-nssm)

**Comment by frdmn** [comment]
That's intended because the mentioned repo is meant "for the start/stop daemons" and not the actual blog post.

**Comment by rnarian** [comment]
Can we close this then?

**Comment by frdmn** [comment]
No, I would like to automate that in the theme somehow so I can get rid of the special reference inside the blog post itself. What do you think?
